"make package" failure on OSX with Qt5.5

Running `make package` fails with:

```
Run CPack packaging tool...
CPack: Create package using DragNDrop
CPack: Install projects
CPack: - Run preinstall target for: Cockatrice
CPack: - Install project: Cockatrice
warning: target '@rpath/QtGui.framework/Versions/5/QtGui' is not absolute...
warning: target '@rpath/QtGui.framework/Versions/5/QtGui' does not exist...
error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/otool: can't open file: @rpath/QtGui.framework/Versions/5/QtGui (No such file or directory)
error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/otool: can't open file: @rpath/QtGui.framework/Versions/5/QtGui (No such file or directory)
warning: target '@rpath/QtCore.framework/Versions/5/QtCore' is not absolute...
warning: target '@rpath/QtCore.framework/Versions/5/QtCore' does not exist...
error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/otool: can't open file: @rpath/QtCore.framework/Versions/5/QtCore (No such file or directory)
error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/otool: can't open file: @rpath/QtCore.framework/Versions/5/QtCore (No such file or directory)
warning: target '@rpath/QtDBus.framework/Versions/5/QtDBus' is not absolute...
warning: target '@rpath/QtDBus.framework/Versions/5/QtDBus' does not exist...
error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/otool: can't open file: @rpath/QtDBus.framework/Versions/5/QtDBus (No such file or directory)
error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/otool: can't open file: @rpath/QtDBus.framework/Versions/5/QtDBus (No such file or directory)
warning: target '@rpath/QtPrintSupport.framework/Versions/5/QtPrintSupport' is not absolute...
warning: target '@rpath/QtPrintSupport.framework/Versions/5/QtPrintSupport' does not exist...
error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/otool: can't open file: @rpath/QtPrintSupport.framework/Versions/5/QtPrintSupport (No such file or directory)
error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/otool: can't open file: @rpath/QtPrintSupport.framework/Versions/5/QtPrintSupport (No such file or directory)
warning: target '@rpath/QtWidgets.framework/Versions/5/QtWidgets' is not absolute...
warning: target '@rpath/QtWidgets.framework/Versions/5/QtWidgets' does not exist...
error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/otool: can't open file: @rpath/QtWidgets.framework/Versions/5/QtWidgets (No such file or directory)
error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/otool: can't open file: @rpath/QtWidgets.framework/Versions/5/QtWidgets (No such file or directory)
error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/otool: can't open file: @rpath/QtCore.framework/Versions/5/QtCore (No such file or directory)
error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/otool: can't open file: @rpath/QtDBus.framework/Versions/5/QtDBus (No such file or directory)
error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/otool: can't open file: @rpath/QtGui.framework/Versions/5/QtGui (No such file or directory)
error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/otool: can't open file: @rpath/QtPrintSupport.framework/Versions/5/QtPrintSupport (No such file or directory)
error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/otool: can't open file: @rpath/QtWidgets.framework/Versions/5/QtWidgets (No such file or directory)
warning: target '@rpath/QtGui.framework/Versions/5/QtGui' is not absolute...
warning: target '@rpath/QtGui.framework/Versions/5/QtGui' does not exist...
error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/otool: can't open file: @rpath/QtGui.framework/Versions/5/QtGui (No such file or directory)
error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/otool: can't open file: @rpath/QtGui.framework/Versions/5/QtGui (No such file or directory)
warning: target '@rpath/QtCore.framework/Versions/5/QtCore' is not absolute...
warning: target '@rpath/QtCore.framework/Versions/5/QtCore' does not exist...
error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/otool: can't open file: @rpath/QtCore.framework/Versions/5/QtCore (No such file or directory)
error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/otool: can't open file: @rpath/QtCore.framework/Versions/5/QtCore (No such file or directory)
warning: target '@rpath/QtDBus.framework/Versions/5/QtDBus' is not absolute...
warning: target '@rpath/QtDBus.framework/Versions/5/QtDBus' does not exist...
error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/otool: can't open file: @rpath/QtDBus.framework/Versions/5/QtDBus (No such file or directory)
error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/otool: can't open file: @rpath/QtDBus.framework/Versions/5/QtDBus (No such file or directory)
warning: target '@rpath/QtGui.framework/Versions/5/QtGui' is not absolute...
warning: target '@rpath/QtGui.framework/Versions/5/QtGui' does not exist...
error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/otool: can't open file: @rpath/QtGui.framework/Versions/5/QtGui (No such file or directory)
error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/otool: can't open file: @rpath/QtGui.framework/Versions/5/QtGui (No such file or directory)
warning: target '@rpath/QtCore.framework/Versions/5/QtCore' is not absolute...
warning: target '@rpath/QtCore.framework/Versions/5/QtCore' does not exist...
error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/otool: can't open file: @rpath/QtCore.framework/Versions/5/QtCore (No such file or directory)
error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/otool: can't open file: @rpath/QtCore.framework/Versions/5/QtCore (No such file or directory)
warning: target '@rpath/QtDBus.framework/Versions/5/QtDBus' is not absolute...
warning: target '@rpath/QtDBus.framework/Versions/5/QtDBus' does not exist...
error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/otool: can't open file: @rpath/QtDBus.framework/Versions/5/QtDBus (No such file or directory)
error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/otool: can't open file: @rpath/QtDBus.framework/Versions/5/QtDBus (No such file or directory)
warning: target '@rpath/QtSql.framework/Versions/5/QtSql' is not absolute...
warning: target '@rpath/QtSql.framework/Versions/5/QtSql' does not exist...
error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/otool: can't open file: @rpath/QtSql.framework/Versions/5/QtSql (No such file or directory)
error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/otool: can't open file: @rpath/QtSql.framework/Versions/5/QtSql (No such file or directory)
warning: target '@rpath/QtCore.framework/Versions/5/QtCore' is not absolute...
warning: target '@rpath/QtCore.framework/Versions/5/QtCore' does not exist...
error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/otool: can't open file: @rpath/QtCore.framework/Versions/5/QtCore (No such file or directory)
error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/otool: can't open file: @rpath/QtCore.framework/Versions/5/QtCore (No such file or directory)
error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/otool: can't open file: @rpath/QtSql.framework/Versions/5/QtSql (No such file or directory)
warning: target '/opt/local/lib/mysql55/mysql/libmysqlclient.18.dylib' does not exist...
error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/otool: can't open file: /opt/local/lib/mysql55/mysql/libmysqlclient.18.dylib (No such file or directory)
error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/otool: can't open file: /opt/local/lib/mysql55/mysql/libmysqlclient.18.dylib (No such file or directory)
warning: target '@rpath/QtSql.framework/Versions/5/QtSql' is not absolute...
warning: target '@rpath/QtSql.framework/Versions/5/QtSql' does not exist...
error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/otool: can't open file: @rpath/QtSql.framework/Versions/5/QtSql (No such file or directory)
error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/otool: can't open file: @rpath/QtSql.framework/Versions/5/QtSql (No such file or directory)
warning: target '@rpath/QtCore.framework/Versions/5/QtCore' is not absolute...
warning: target '@rpath/QtCore.framework/Versions/5/QtCore' does not exist...
error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/otool: can't open file: @rpath/QtCore.framework/Versions/5/QtCore (No such file or directory)
error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/otool: can't open file: @rpath/QtCore.framework/Versions/5/QtCore (No such file or directory)
error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/otool: can't open file: /opt/local/lib/mysql55/mysql/libmysqlclient.18.dylib (No such file or directory)
warning: target '@rpath/QtSql.framework/Versions/5/QtSql' is not absolute...
warning: target '@rpath/QtSql.framework/Versions/5/QtSql' does not exist...
error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/otool: can't open file: @rpath/QtSql.framework/Versions/5/QtSql (No such file or directory)
error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/otool: can't open file: @rpath/QtSql.framework/Versions/5/QtSql (No such file or directory)
warning: target '@rpath/QtCore.framework/Versions/5/QtCore' is not absolute...
warning: target '@rpath/QtCore.framework/Versions/5/QtCore' does not exist...
error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/otool: can't open file: @rpath/QtCore.framework/Versions/5/QtCore (No such file or directory)
error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/otool: can't open file: @rpath/QtCore.framework/Versions/5/QtCore (No such file or directory)
warning: target '@rpath/QtSql.framework/Versions/5/QtSql' is not absolute...
warning: target '@rpath/QtSql.framework/Versions/5/QtSql' does not exist...
error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/otool: can't open file: @rpath/QtSql.framework/Versions/5/QtSql (No such file or directory)
error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/otool: can't open file: @rpath/QtSql.framework/Versions/5/QtSql (No such file or directory)
warning: target '@rpath/QtCore.framework/Versions/5/QtCore' is not absolute...
warning: target '@rpath/QtCore.framework/Versions/5/QtCore' does not exist...
error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/otool: can't open file: @rpath/QtCore.framework/Versions/5/QtCore (No such file or directory)
error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/otool: can't open file: @rpath/QtCore.framework/Versions/5/QtCore (No such file or directory)
warning: target '@rpath/QtWidgets.framework/Versions/5/QtWidgets' is not absolute...
warning: target '@rpath/QtWidgets.framework/Versions/5/QtWidgets' does not exist...
error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/otool: can't open file: @rpath/QtWidgets.framework/Versions/5/QtWidgets (No such file or directory)
error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/otool: can't open file: @rpath/QtWidgets.framework/Versions/5/QtWidgets (No such file or directory)
warning: target '@rpath/QtNetwork.framework/Versions/5/QtNetwork' is not absolute...
warning: target '@rpath/QtNetwork.framework/Versions/5/QtNetwork' does not exist...
error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/otool: can't open file: @rpath/QtNetwork.framework/Versions/5/QtNetwork (No such file or directory)
error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/otool: can't open file: @rpath/QtNetwork.framework/Versions/5/QtNetwork (No such file or directory)
warning: target '@rpath/QtSql.framework/Versions/5/QtSql' is not absolute...
warning: target '@rpath/QtSql.framework/Versions/5/QtSql' does not exist...
error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/otool: can't open file: @rpath/QtSql.framework/Versions/5/QtSql (No such file or directory)
error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/otool: can't open file: @rpath/QtSql.framework/Versions/5/QtSql (No such file or directory)
warning: target '@rpath/QtGui.framework/Versions/5/QtGui' is not absolute...
warning: target '@rpath/QtGui.framework/Versions/5/QtGui' does not exist...
error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/otool: can't open file: @rpath/QtGui.framework/Versions/5/QtGui (No such file or directory)
error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/otool: can't open file: @rpath/QtGui.framework/Versions/5/QtGui (No such file or directory)
warning: target '@rpath/QtCore.framework/Versions/5/QtCore' is not absolute...
warning: target '@rpath/QtCore.framework/Versions/5/QtCore' does not exist...
error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/otool: can't open file: @rpath/QtCore.framework/Versions/5/QtCore (No such file or directory)
error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/otool: can't open file: @rpath/QtCore.framework/Versions/5/QtCore (No such file or directory)
error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/otool: can't open file: @rpath/QtNetwork.framework/Versions/5/QtNetwork (No such file or directory)
Error copying file "@rpath/QtCore.framework/Versions/5/QtCore" to "/Users/daenyth/Cockatrice/build/_CPack_Packages/OSX/DragNDrop/Cockatrice-osx_git-1409b97_2015-09-18/servatrice.app/Contents/Frameworks/QtCore.framework/Versions/5/QtCore".
Error copying file "@rpath/QtDBus.framework/Versions/5/QtDBus" to "/Users/daenyth/Cockatrice/build/_CPack_Packages/OSX/DragNDrop/Cockatrice-osx_git-1409b97_2015-09-18/servatrice.app/Contents/Frameworks/QtDBus.framework/Versions/5/QtDBus".
Error copying file "@rpath/QtGui.framework/Versions/5/QtGui" to "/Users/daenyth/Cockatrice/build/_CPack_Packages/OSX/DragNDrop/Cockatrice-osx_git-1409b97_2015-09-18/servatrice.app/Contents/Frameworks/QtGui.framework/Versions/5/QtGui".
Error copying file "@rpath/QtPrintSupport.framework/Versions/5/QtPrintSupport" to "/Users/daenyth/Cockatrice/build/_CPack_Packages/OSX/DragNDrop/Cockatrice-osx_git-1409b97_2015-09-18/servatrice.app/Contents/Frameworks/QtPrintSupport.framework/Versions/5/QtPrintSupport".
Error copying file "@rpath/QtWidgets.framework/Versions/5/QtWidgets" to "/Users/daenyth/Cockatrice/build/_CPack_Packages/OSX/DragNDrop/Cockatrice-osx_git-1409b97_2015-09-18/servatrice.app/Contents/Frameworks/QtWidgets.framework/Versions/5/QtWidgets".
Error copying file "@rpath/QtSql.framework/Versions/5/QtSql" to "/Users/daenyth/Cockatrice/build/_CPack_Packages/OSX/DragNDrop/Cockatrice-osx_git-1409b97_2015-09-18/servatrice.app/Contents/Frameworks/QtSql.framework/Versions/5/QtSql".
Error copying file "/opt/local/lib/mysql55/mysql/libmysqlclient.18.dylib" to "/Users/daenyth/Cockatrice/build/_CPack_Packages/OSX/DragNDrop/Cockatrice-osx_git-1409b97_2015-09-18/servatrice.app/Contents/MacOS/libmysqlclient.18.dylib".
Error copying file "@rpath/QtNetwork.framework/Versions/5/QtNetwork" to "/Users/daenyth/Cockatrice/build/_CPack_Packages/OSX/DragNDrop/Cockatrice-osx_git-1409b97_2015-09-18/servatrice.app/Contents/Frameworks/QtNetwork.framework/Versions/5/QtNetwork".
warning: target '/Users/daenyth/Cockatrice/build/_CPack_Packages/OSX/DragNDrop/Cockatrice-osx_git-1409b97_2015-09-18/servatrice.app/Contents/Frameworks/QtCore.framework/Versions/5/QtCore' does not exist...
error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/otool: can't open file: /Users/daenyth/Cockatrice/build/_CPack_Packages/OSX/DragNDrop/Cockatrice-osx_git-1409b97_2015-09-18/servatrice.app/Contents/Frameworks/QtCore.framework/Versions/5/QtCore (No such file or directory)
error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/otool: can't open file: /Users/daenyth/Cockatrice/build/_CPack_Packages/OSX/DragNDrop/Cockatrice-osx_git-1409b97_2015-09-18/servatrice.app/Contents/Frameworks/QtCore.framework/Versions/5/QtCore (No such file or directory)
chmod: /Users/daenyth/Cockatrice/build/_CPack_Packages/OSX/DragNDrop/Cockatrice-osx_git-1409b97_2015-09-18/servatrice.app/Contents/Frameworks/QtCore.framework/Versions/5/QtCore: No such file or directory
error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/install_name_tool: can't open file: /Users/daenyth/Cockatrice/build/_CPack_Packages/OSX/DragNDrop/Cockatrice-osx_git-1409b97_2015-09-18/servatrice.app/Contents/Frameworks/QtCore.framework/Versions/5/QtCore (No such file or directory)
CMake Error at /usr/local/Cellar/cmake/3.3.0/share/cmake/Modules/BundleUtilities.cmake:787 (message):
  Command failed:

   'install_name_tool' '-id' '@executable_path/../Frameworks/QtCore.framework/Versions/5/QtCore' '/Users/daenyth/Cockatrice/build/_CPack_Packages/OSX/DragNDrop/Cockatrice-osx_git-1409b97_2015-09-18/servatrice.app/Contents/Frameworks/QtCore.framework/Versions/5/QtCore'
Call Stack (most recent call first):
  /usr/local/Cellar/cmake/3.3.0/share/cmake/Modules/BundleUtilities.cmake:848 (fixup_bundle_item)
  /Users/daenyth/Cockatrice/build/servatrice/cmake_install.cmake:66 (fixup_bundle)
  /Users/daenyth/Cockatrice/build/cmake_install.cmake:33 (include)


CPack Error: Error when generating package: Cockatrice
make: *** [package] Error 1
```

Shell history:
```bash
cmake .. -DCMAKE_PREFIX_PATH=~/Qt/5.5/clang_64/
make -j8
make package
```

Cmake output
```
UPDATE TRANSLATIONS: OFF
-- Found Qt 5.5.0
-- Configuring done
-- Generating done
-- Build files have been written to: /Users/daenyth/Cockatrice/build
```

--- a/cockatrice/CMakeLists.txt
+++ b/cockatrice/CMakeLists.txt
@@ -281,6 +281,7 @@ if(APPLE)
     # these needs to be relative to CMAKE_INSTALL_PREFIX
     set(plugin_dest_dir cockatrice.app/Contents/Plugins)
     set(qtconf_dest_dir cockatrice.app/Contents/Resources)
+    get_filename_component(QT_LIBRARY_DIR "${QT_LIBRARY_DIR}/.." ABSOLUTE)
 
     # qt4: codecs, iconengines, imageformats
     # qt5: audio, iconengines, imageformats, platforms, printsupport

--- a/oracle/CMakeLists.txt
+++ b/oracle/CMakeLists.txt
@@ -185,6 +185,7 @@ if(APPLE)
     # these needs to be relative to CMAKE_INSTALL_PREFIX
     set(plugin_dest_dir oracle.app/Contents/Plugins)
     set(qtconf_dest_dir oracle.app/Contents/Resources)
+    get_filename_component(QT_LIBRARY_DIR "${QT_LIBRARY_DIR}/.." ABSOLUTE)
 
     # qt4: codecs, iconengines, imageformats
     # qt5: iconengines, platforms

--- a/servatrice/CMakeLists.txt
+++ b/servatrice/CMakeLists.txt
@@ -146,6 +146,7 @@ if(APPLE)
     # these needs to be relative to CMAKE_INSTALL_PREFIX
     set(plugin_dest_dir servatrice.app/Contents/Plugins)
     set(qtconf_dest_dir servatrice.app/Contents/Resources)
+    get_filename_component(QT_LIBRARY_DIR "${QT_LIBRARY_DIR}/.." ABSOLUTE)
 
     # qt4: codecs, sqldrivers
     # qt5: platforms, sqldrivers

